.github/CODEOWNERS

```diff
@@ -7,15 +7,26 @@
 # For documentation on this file, see https://help.github.com/articles/about-codeowners/
 # Mentioned users will get code review requests.
 
+# Boostraping and core infra
+pkgs/stdenv/	@Ericson2314
+pkgs/build-support/cc-wrapper/	@Ericson2314
+
+# Libraries
+lib/  @Profpatsch
+
+# Build-Support
+pkgs/build-support  @Profpatsch
```

Comment:
Last matching pattern counts, so this should go before my cc-wrapper one

Reply:
Ah, sorry. Will fix in a new Pr.